packages/flutter_tools/lib/src/test/test_wrapper.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:test_api/backend.dart'; // ignore: deprecated_member_use
+import 'package:test_core/src/runner/platform.dart'; // ignore: implementation_imports
+import 'package:test_core/src/executable.dart' as _test; // ignore: implementation_imports
+import 'package:test_core/src/runner/hack_register_platform.dart' as _hack; // ignore: implementation_imports
+
+import '../base/context.dart';
+
+export 'package:test_api/backend.dart' show Runtime; // ignore: deprecated_member_use
+
+Test get test => context.get<Test>() ?? const PackageTestTest();
```

Comment:
The API presented by the wrapper can hide the distinction between "test" and "hack". Why not have just one class here? That let's you consider exposing the hack as optional named parameters to test.main() instead of a separate call, for example.

Reply:
Done